File: week-02/day-01/draw_pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was
a = int(input())
for i in range(1,a,2):
    print(' ' * (i-1) + '*' *i)

File: week-07/day-01/draw-pyramid.js
'use strict';

var lineCount = 4;
let star = '*'

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

let a = lineCount - 1;

for (let i = lineCount; i > 0; i--) {
  console.log(' '.repeat(i) + star)
  star += '**'
}
